samples/cpp/benchmark_app/main.cpp

```diff
@@ -377,20 +360,22 @@ int main(int argc, char* argv[]) {
         // Update config per device according to command line parameters
         for (auto& device : devices) {
             auto& device_config = config[device];
-
-            // high-level performance modes
-            if (!device_config.count(ov::hint::performance_mode.name())) {
-                device_config.emplace(ov::hint::performance_mode(get_performance_hint(device, core)));
+            auto ov_perf_hint = get_performance_hint(device, core);
+            if (FLAGS_hint != "") {
```

Comment:
By default hint is "". I suppose we have to check `isFlagSetInCommandLine`

Reply:
Updated. Thanks.